Doc/tutorial/modules.rst

```diff
@@ -112,6 +112,17 @@ Note that in general the practice of importing ``*`` from a module or package is
 frowned upon, since it often causes poorly readable code. However, it is okay to
 use it to save typing in interactive sessions.
 
+If as a user of the module you would like to import a module and make it available
+under a different name within the current namespace you can use ``import as``.
```

Comment:
The "[...] you can use \`\`import as\`\`" part can be confusing since `import as` is not valid Python.

I'd use the following sentence as a base in `Doc/reference/simple_stmts.rst`:

```
* If the module name is followed by :keyword:`as`, then the name
  following :keyword:`as` is bound directly to the imported module.
```

Reply:
Also, please end the sentence with `::` (or add it not the next line):

```rst
[...] you can use ``import as``::

   >>> import fibo as fib
   ...
```

or

```rst
[...] you can use ``import as``.

::

   >>> import fibo as fib
   ...
```